packages/liquidator/test/Liquidator.js

```diff
@@ -964,6 +973,157 @@ contract("Liquidator.js", function(accounts) {
           });
         });
       });
+      describe("enabling withdraw defense feature", () => {
+        it("should initilize when enabled", async () => {
+          const liquidatorConfig = {
+            whaleDefenseFundWei: 1,
+            defenseActivationPercent: 50
+          };
+          const liquidator = new Liquidator({
+            logger: spyLogger,
+            expiringMultiPartyClient: empClient,
+            gasEstimator,
+            votingContract: mockOracle.contract,
+            syntheticToken: syntheticToken.contract,
+            priceFeed: priceFeedMock,
+            account: accounts[0],
+            empProps,
+            config: liquidatorConfig
+          });
+          assert.ok(liquidator);
```

Comment:
that's some new syntax! pretty clean. Few other places we could validate no errors like this.

Reply:
Does this assert check for errors? Or just for truthiness? I thought this was just another way of writing `assert()` or `assert.isTrue()`?

Reply:
i think its the same thing as assert. Not sure why I wrote it that way